cupy/random/_generator.py

```diff
@@ -143,6 +143,7 @@ def dirichlet(self, alpha, size=None, dtype=float):
         if size is None:
             size = alpha.shape
         else:
+            size = list(size)  # In case size is an int
```

Comment:
`list(5)` causes `TypeError: 'int' object is not iterable`.  How about aligning with this?

https://github.com/cupy/cupy/blob/79085bcdb5ebfecf906a8bc4453dfa0f85bbddb4/cupy/random/_generator.py#L660-L661

```suggestion
            if isinstance(size, int):
                size = size,
```

Reply:
Can do even better by matching multivariate_normal, supporting both `int` and `cupy.integer`. Just pushed that fix.

Why isn't my test case being run, to catch this error?

Reply:
> Why isn't my test case being run, to catch this error?

It's best to test locally. Tests are not run until a reviewer launches CuPy's CI. The GH action etc just test for building and formatting.